code/src/components/AddNewTask.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { tasks } from 'reducers/tasks'
+import "./AddNewTask.css"
+
+export const AddNewTask = () => {
+  const [newTask, setNewTask] = useState('')
+  const dispatch = useDispatch()
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    dispatch(tasks.actions.addTask(newTask))
+    setNewTask('')
```

Comment:
To make this even better, think about handling an empty input. Right now it's possible to save a blank todo.
Maybe add something like:
`if (!newTask) return`
to make sure to return from the function if there's no new task written